source/openapi-spec.yml

```diff
@@ -242,6 +278,12 @@ components:
     Qualification:
       type: object
       additionalProperties: false
+      required:
+        - qualification_type
+        - subject
+        - result
+        - award_year
+        - place_of_study
```

Comment:
What about 

`awarding_body_name`, `awarding_body_country` and `equivalency_details`?

Reply:
I thought `awarding_body_name` and `awarding_body_country` were optional?

`equivalency_details` could be only present if it's an international qualification?

Reply:
I have an unexamined assumption that it is simpler to always return the same structure with nulls in it than to return a reduced structure without nulls. Happy to go with your instinct to leave these out